build-support/bin/native/bootstrap_code.sh

```diff
@@ -62,7 +62,6 @@ function _build_native_code() {
   # Builds the native code, and echos the path of the built binary.
 
   (
-    cd "${REPO_ROOT}"
```

Comment:
This edit broke using pants from other repos... was it necessary for something here?

Reply:
Aha! No, it wasn't necessary; this `cd` used to do two things (ensure we're in a child of the directory containing `.cargo`, and ensure we're a child of the directory containing `rust-toolchain`); I removed it because the first is now no longer necessary, forgetting that the second is. Will re-add with a comment.